frame/support/src/traits/tokens/nonfungible.rs

```diff
@@ -65,11 +65,14 @@ pub trait Inspect<AccountId> {
 /// Interface for enumerating items in existence or owned by a given account over a collection
 /// of NFTs.
 pub trait InspectEnumerable<AccountId>: Inspect<AccountId> {
+	type ItemsIter: Iterator<Item = Self::ItemId>;
+	type OwnedIter: Iterator<Item = Self::ItemId>;
```

Comment:
Please add some docs.

Reply:
I've added docs to both of these, however they're quite basic - I wasn't quite sure what to write. Let me know if you think they should be expanded (and how :sweat_smile:)